packages/discord.js/src/structures/Poll.js

```diff
@@ -79,6 +108,21 @@ class Poll extends Base {
         const answer = this.answers.get(answerResult.id);
         answer?._patch(answerResult);
       }
+    } else if (data.question) {
```

Comment:
I don't believe these should be `else if`s, but rather regular `if`s after the previous if-else that was here